src/app/story/storyrow/totaltimes/totaltimes.component.css

```diff
@@ -0,0 +1,13 @@
+@media (max-width: 991px) {
```

Comment:
Optional: In this project, we try to follow the bootstrap model of applying styles to mobile by default and then making changes to larger screens, not the other way around. For example, to follow the rest of the code base, you would do:

```css
.mobile-hide {
    display: none;
}

@media (min-width: 992px) {
    .mobile-hide {
        display: initial;
    }
}
```